ci-operator/step-registry/ipi/conf/mirror/ipi-conf-mirror-commands.sh

```diff
@@ -7,13 +7,13 @@ set -o pipefail
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
 # mirror registry
-install_config_icsp_patch="${SHARED_DIR}/install-config-icsp.yaml.patch"
```

Comment:
@aleskandro I saw several baremetal-lab steps are referencing the patch file name, now I am renaming it, can you help evaluate the impact on the baremetal install step? Or what change need to do on the baremetal-lab steps?

Reply:
We mostly want to catch the existence of a file that can inform us about the mirroring of the payload. From a quick look, you are mostly changing the name to enable the posibility of using IDMS instead of ICSP. Could you change the name in file like https://github.com/openshift/release/blob/f452d3d8df9a9b9d585e6361ac9e2691afd052d8/ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh#L62 to reflect this change? It should be enough

Reply:
Got it, easy fix, wait a sec.

Reply:
@aleskandro updated, pls help review again, thanks.

Reply:
/lgtm
Thanks @jianlinliu